pilot/adapter/config/ingress/controller_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/kubernetes/fake"
 
-	proxyconfig "istio.io/api/proxy/v1/config"
+	proxyconfig "istio.io/api/routing/v1alpha1"
```

Comment:
`routingconfig` or maybe even `routing` to match the unaliased package name?

Reply:
Done, went with `routing`.